### PR TITLE
add eventhandler to quickly navigate through plugin list with keybord via keyup or keydown

### PR DESCRIPTION
**Pull request by manpenaloza**
I added some comfortable (for now browser-vanilla) possibility to navigate through all listed PluginListItems of a PluginList via keyup and keydown press. In case you like it and want to use it too, feel free to pull.

### CODE REVIEW

File: js/components/App/index.js [hunk 1]
```diff
@@ -17,23 +17,58 @@ class App extends Component {
...
+            case 38:
+              if (this.keyUpDownCounter !== 1 && this.keyUpDownCounter !== 0) {
+                this.keyUpDownCounter--;
+                pluginNodes[this.keyUpDownCounter - 1].focus();
+              }
+              break;
+          }
+        }
+      }, false);
+    }
```
Comment (browniefed) [inline js/components/App/index.js:56]: Could you please use React and not this.

File: js/components/App/index.js [hunk 2]
```diff
@@ -17,23 +17,58 @@ class App extends Component {
...
     this.props.router[action](`${pathname}?searchTerm=${evt.target.value}`)
-	}
+    this.keyUpDownCounter = 0;
+  }
 
 	render() {
     const { children, location: { query: { searchTerm }, pathname } } = this.props;
     const tag = getTag(pathname);
     let content = children;
+    this.keyUpDownCounter = 0;
```
Comment (browniefed) [inline js/components/App/index.js:71]: Never mutate in render.

**Comment by browniefed** [comment]
Thank you for the contribution, it'd be great to switch over to use a Declarative React component to handle key events.

**Comment by manpenaloza** [comment]
thx for feedback, just found that https://facebook.github.io/react/docs/events.html#keyboard-events and will go with that + rm mutation in render. pr follows.

**Comment by manpenaloza** [comment]
I'll come up with a completely new Fork/PR to solve this and close this one.

**Pull request closed without merging**